packages/botonic-core/src/models/events/message/carousel.ts

```diff
@@ -0,0 +1,8 @@
+import { BotonicMessageEvent, MessageEventType } from '.'
+import { CarouselElement } from './element'
+
+export interface CarouselMessageEvent
+  extends BotonicMessageEvent<MessageEventType.carousel> {
+  type: MessageEventType.carousel
```

Comment:
Since `type` is already defined in `BotonicMessageEvent` is not needed to be redefined here, right?
This is happening also with `CustomMessageEvent`.

Reply:
yes, that was right before the last commit. With the new approach we need to explicitly add it in order to let typescript tell between the different "BotonicEvents".